deploy/nginx.conf

```diff
@@ -137,22 +137,22 @@ http {
 
       # Configure error pages
 
-      error_page 404 /errors/404/index.html;
-      error_page 400 401 402 403 405 406 407 408 409 410 411 412 413 414 415 416 417 418 420 422 423 424 426 428 429 431 444 449 450 451 500 501 502 503 504 505 506 507 508 509 510 511 /errors/generic/index.html;
+      error_page 404 /404.html;
+      error_page 400 401 402 403 405 406 407 408 409 410 411 412 413 414 415 416 417 418 420 422 423 424 426 428 429 431 444 449 450 451 500 501 502 503 504 505 506 507 508 509 510 511 /errors/generic.html;
```

Comment:
If this requires a separate Nginx reload/restart/kick we can duplicate the page instead?

E.g. One 404 page could `{% include "errors/404.njk" %}` the other

Reply:
When we deploy to CloudFoundry (PaaS) an entirely new container is spun up, then the route is reassigned to the new container once it is healthy and serving traffic. So effectively it's a brand new instance of nginx.

If we want to test it, we could manually deploy to a separate PaaS instance?

Reply:
I think I'm happy knowing the pages in the config still exist, should be fine:

https://deploy-preview-3309--govuk-design-system-preview.netlify.app/404.html
https://deploy-preview-3309--govuk-design-system-preview.netlify.app/errors/generic.html

It wasn't strictly necessary to use `permalink: false` in **generic.njk** but they both match now

Just means it's moved to ~`/errors/generic/index.html`~ `/errors/generic.html`

Reply:
@36degrees Shall we get this merged?